combinations/hash.tsv

```diff
@@ -551,3 +551,4 @@ r-hwriter=1.3.2,r-ggplot2=3.4.4,r-ini=0.3.1,r-ashr=2.2_63,r-dplyr=1.1.4,r-ggrepe
 pandas=2.2.1,tensorflow=2.15.0,scikit-learn=1.4.1,umap-learn=0.5.5,matplotlib=3.8.0
 magus-msa=0.1.3,pigz=2.8
 gatk4=4.5.0.0,samtools=1.19.2,bcftools=1.19,perl=5.32.1,r-base=4.3
+xengsort=2.0.5,pigz=2.8,htop=3.2.2,seqkit=2.7.0,csvtk=0.29.0
```

Comment:
htop? This should not be needed by your workflow isn't it?

Reply:
It is needed for debugging when the job runs. But if that is a show-stopper for this, by all means we can remove that.'

Do you want me to update